tests/functional/verification_experiments.js

```diff
@@ -1,55 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-define([
-  'intern',
-  'intern!object',
-  'tests/functional/lib/helpers'
-], function (intern, registerSuite, FunctionalHelpers) {
-  var EXP_MAILCHECK_URL = intern.config.fxaContentRoot + 'signup?forceExperiment=mailcheck&automatedBrowser=true';
```

Comment:
@divyabiyani for this file, we should keep it, but rename it to `mailcheck` (make sure to rename it in the list of functional tests as well).

We can remove the `'signup?forceExperiment=mailcheck&automatedBrowser=true'` query part and only leave the `'treatment works': function () {` test. Delete the `'control works': function () {` test though